policy-addon-kubernetes/helm-charts/index.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+entries:
+  azure-policy-addon-aks-engine:
+  - apiVersion: v1
+    appVersion: "1"
+    created: "2019-10-22T10:39:56.7851001-07:00"
+    description: Helm chart for deploying Azure Policy add-on in AKS Engine
+    digest: b20aad1f43e90211b5541ed5c7d14458086685f50899ffdb883eb35fe1728da4
+    home: https://aka.ms/kubepolicydoc
+    keywords:
+    - azure
+    - azure policy
+    - azure-policy
+    - gatekeeper
+    - governance
+    - compliance
+    - aks engine
+    - aks-engine
+    - kubernetes
+    maintainers:
+    - email: azuredg@microsoft.com
+      name: azuredg
+    name: azure-policy-addon-aks-engine
+    sources:
+    - https://github.com/Azure/azure-policy-kubernetes
```

Comment:
Do we want to link to this repo when it's private?

Reply:
Sources is optional, I will remove it.